lib/ui/recommendation.js

```diff
@@ -167,6 +167,12 @@ Recommendation.prototype = {
       return;
     }
 
+    // For metrics, record whether the event was a click or key.
+    const eventData = {
+      click: !!evt.button,
+      type: this.resultType
+    };
+
```

Comment:
The changes in this file are stale (from an earlier attempt), ignore. I'll remove.